frontend/src/constants/resources.ts

```diff
@@ -284,6 +284,11 @@ export const BOARD_CONFIG_INFO_TITLE = {
   EMPTY: '',
 };
 
+export const PIPELINE_CONFIG_INFO_TITLE = {
```

Comment:
this variable like an `enum`, please covert it to an enum, and rename it as `PipelineConfigInfoTitle`

ref to https://zhuanlan.zhihu.com/p/88609380, same as JS